Ullaakut/cameradar
# Support multi-route cameras & other improvements

## Goal of this PR

Fixes #264 
Fixes #265 
Fixes #266

It introduces Cameradar v5, with the following changes:

* Support multi-route detection
    * Change stream model to support multiple routes
    * Simplify attack algorithm
    * Use dummy route to detect cameras which accept all routes
* Move issue template to `.github` and remove old changelog file
* Fix stream discovery on non-standard ports by using service info mode in nmap scan
* Reword error message when no stream is found
* Remove unnecessary import alias
* Set module version to v5
    * Update README to reflect changes
    * Various README improvements
* No longer enables debug by default
* Makes verbose mode work (used to not do anything)

## How to test it

* Verify that it can detect streams on a non-standard port (like 8555 for example)
* Verify that it can detect streams on a camera with multiple stream routes
* Verify that it can be used with the go module `github.com/Ullaakut/cameradar/v5`

## Screenshot

<img width="649" alt="Screenshot 2020-05-04 at 12 31 34 AM" src="https://user-images.githubusercontent.com/6976628/80938885-03d70980-8ddb-11ea-9e31-49ea316afbfc.png">
